runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -819,8 +820,12 @@ public synchronized void invalidateLayout() {
             // Don't create a new request for a layout if there is one pending.
             return;
         }
-        layout = fetchLayout(latestLayout == null
-                ? bootstrapLayoutServers : latestLayout.getLayoutServers());
+
```

Comment:
Create a separate PR for code cleanup.
The subject of this PR says "reduce server logs".

Reply:
Creating a PR, then running it several times then asking many times to review changes is a pretty heavy loaded and time-consuming operation. I don't see any reasons for extracting this small piece of code into another pr. Also, since I changed the related method (fetchLayout) I believe it's ok to bring this small change in this pr.

Reply:
Put a note in the git commit.